adding issue template

Samiksha-bajoria opened the pull request:
Issue Template is added for bug report, feature request, documentation, custom issue and all are connected by config.yml.

fixes #10

Samiksha-bajoria committed:
Create bug_report.md

Samiksha-bajoria committed:
Create feature_request.md

Samiksha-bajoria committed:
Create documentation_issue.md

Samiksha-bajoria committed:
Create custom_issue.md

Samiksha-bajoria committed:
Create config.yml

RayBreeze reviewed (changes requested):
Dear @Samiksha-bajoria,
Thank you for your contributions in SysMood. Can you please remove the emojis from the issue templates and also I think that these issue files are too large and. please ensure to make it a less complicated. It will be helpful for developers then. Thank you 

Regards,
Samman

Samiksha-bajoria commented:
Ok will make the changes asap .

Samiksha-bajoria committed:
Refactor bug report template for better usability

Updated the bug report template to improve clarity and structure.

Samiksha-bajoria committed:
Revise feature request template structure

Updated the feature request template to streamline sections and improve clarity.

Samiksha-bajoria committed:
Revise documentation issue template format

Updated the documentation issue template to clarify sections and improve structure.

Samiksha-bajoria committed:
Update custom_issue.md

RayBreeze reviewed (approved):
Hello @Samiksha-bajoria,
Thank you for your contribution in SysMood. These small documentation change will surely help us for better maintenance of the project for me and the fellow contributors. Thanks for making SysMood better. Please don't forget to add a star and watch for future releases. Love to see more contributions in future...🙂

Regards,
Samman
